src/shared/components/sidebar/_sidebar.scss

```diff
@@ -0,0 +1,93 @@
+.sidebar {
+  height: 100vh;
+  background-color: rgb(240, 240, 240);
+  display: flex;
+  flex-direction: column;
+  transition: width 0.3s ease;
+  position: relative;
+
+  &-button {
+    height: 26px;
+    width: 26px;
+    border-radius: 50%;
+    border: none;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.3);
+    position: absolute;
+    right: -13px;
+    top: 60px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      background-color: white;
+      cursor: pointer;
+    }
+  }
+
+  &.expanded {
+    width: 250px;
+  }
+
+  &.collapsed {
+    width: 65px;
+  }
+
+  &-top {
+    padding: 10px;
+
+    &-user {
+      height: 50px;
+      padding: 10px;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      background-color: white;
+      border-radius: 25px;
+      box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.2);
+      gap: 30px;
+      overflow: hidden;
+
+      &-icon {
+      }
+
+      &-text {
+      }
```

Comment:
Can you delete it, or in future we will have some styles for it?

Reply:
Done